source/install/install-kubernetes-aks.rst

```diff
@@ -0,0 +1,114 @@
+Install Mattermost on Azure AKS
+================================
+
+.. include:: ../_static/badges/allplans-selfhosted.rst
+  :start-after: :nosearch:
+
+You can use a supported `Azure Marketplace Container Offer <https://azuremarketplace.microsoft.com/en-us/marketplace/apps/mattermost.mattermost-operator>`__ to install Mattermost on your existing Azure infrastructure.
+
+Infrastructure pre-requisites
+-----------------------------
+
+Deploying Mattermost on Azure AKS requires the following database and cluster prerequisites.
+
+PostgreSQL v13.0+ database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost requires a pre-existing PostgreSQL database within your infrastructure. We recommend using `Azure Database for PostgreSQL flexible server <https://learn.microsoft.com/en-us/azure/postgresql/>`_. Deploy one by following `this Microsoft quick start guide <https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/quickstart-create-server-portal>`_.
+
+.. tip::
+  We recommend using Private Access for your database.
+
+Running AKS cluster
+~~~~~~~~~~~~~~~~~~~
+
+Mattermost Azure Container Offer requires a pre-existing Kubernetes Cluster with an Ingress Controller pre-installed. We recommend creating a new AKS cluster with the `AGIC add-on enabled <https://learn.microsoft.com/en-us/azure/application-gateway/ingress-controller-overview>`_. 
+
+Follow `this tutorial <https://learn.microsoft.com/en-us/azure/application-gateway/tutorial-ingress-controller-add-on-new>`_ to create a new AKS cluster with the add-on enabled.
+
+.. tip::
+
+  - Connectivity should be already in place between the AKS cluster and the PostgreSQL database.
+  - Any pre-installed Ingress Controller within the cluster that supports the Ingress Kubernetes resource and TLS termination should work out of the box.
+
+Deployment pre-requisites
+-------------------------
+
+Deploying Mattermost on Azure AKS requires the following deployment prerequisites.
+
+Valid DNS name and TLS certificates 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost relies on strong TLS certification in order to provide all the features to users. You need to have access to a DNS zone and be able to provide a valid TLS key and certificate for the Ingress Controller.
+
+Mattermost License and AKS Capacity  
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../_static/badges/ent-only.rst
+  :start-after: :nosearch:
+
+If your deployment option is for more than ``100 users``, you must have more than 2 nodes on your AKS cluster to support High Availability, and you must provide a valid Mattermost License file.
```

Comment:
Though we don't have scaling info yet using Azure, I wonder if we should refer them (temporarily) to https://docs.mattermost.com/guides/scale-mattermost.html so they could have some guidance (by converting AWS resource to Azure).

Reply:
It's not about scaling as such but giving the option to do so . We can limit the installation to some users if we want to but if someone purchases a license then they should be able to scale a bit.